### DISCUSSION TITLE
How to save scripts on this page?

### DISCUSSION THREAD

**Discussion by Droyk** [gildas-lormeau/SingleFile#948] [General]
I'm on Microsoft Edge
Version 99.0.1150.46 (Official build) (64-bit)

after disabling this option
![image](https://user-images.githubusercontent.com/15848350/159241428-01260128-6ecd-4983-b2c9-5e58b76917ac.png)

I'm getting this error 
![image](https://user-images.githubusercontent.com/15848350/159241483-8fddbae1-6050-4619-9919-d3eff9cf387e.png)

here is the saved file 
[Microsoft Edge (3_21_2022 3_36_39 PM).gz](https://github.com/gildas-lormeau/SingleFile/files/8314611/Microsoft.Edge.3_21_2022.3_36_39.PM.gz)
change the extension to `.7z` github doesn't support 7.zip

the reason I disabled the option is because I wanted to save the content of these buttons 
![vWO2bSKgCm](https://user-images.githubusercontent.com/15848350/159243575-92a8f6e9-b88e-485c-9300-b7efd6f53244.png)

here's the website https://www.microsoft.com/en-us/edge/microsoft-365?exp=e01&form=MT005D
I want to save this completely as it is so I can remember the design and the animations of it!

Can the single file help me achieving what I want? if not can anyone please tell me how I can save it?

**Comment by gildas-lormeau** [comment]
I confirm that the saved page is not working properly. Unfortunately, the support of embedded scripts is limited in SingleFile. Did you try to use HTTrack?

**Comment by Droyk** [comment]
Hey, apologies for the late reply I did tried it but it gave me an error it's not user-friendly at all! I need some pro-level assistance to make it work! any other alternative that works out of the box? I did like what HTTrack is offering but it seems like it is made for developers in mind, not your basic savvy users!  anything that is fairly straightforward like Single file but also provides what Httrack does?
